.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - master
```

Comment:
There is no _master_ branch - only _main_

Reply:
indeed, copy and paste error, fixed